src/chainstate/burn/db/sortdb.rs

```diff
@@ -2915,6 +2927,21 @@ impl SortitionDB {
         Ok(())
     }
 
+    fn apply_schema_5(tx: &DBTx, epochs: &[StacksEpoch]) -> Result<(), db_error> {
+        for sql_exec in SORTITION_DB_SCHEMA_5 {
```

Comment:
Per the previous comments, there might need to be a comment here (or perhaps in `SORTITION_DB_SCHEMA_5`) that we are _replacing_ the contents of the `epochs` table.  This is less error-prone than trying to patch it, since the `epochs` table represents ranges of block heights (so, adding the row for epoch 2.2 it would require modifying the row for epoch 2.1 to specify its end height).  This code here achieves the same end, but without explicit consideration for which epoch is being added or modified.

Reply:
Sure, I'll add a comment.

Reply:
Added in [d7827f7](https://github.com/stacks-network/stacks-blockchain/pull/3677/commits/d7827f7cfd20346b67055a3330d4dfbdfa8a0c07)